geopandas/base.py

```diff
@@ -217,51 +289,153 @@ def geom_equals_exact(self, other, tolerance):
+        `interior` of the other but does not contain it, and the dimension of
+        the intersection is less than the dimension of the one or the other.
+
+        Parameters
+        ----------
+        other : GeoSeries or geometric object
+            The GeoSeries (elementwise) or geometric object to test if is
+            crossed.
+        """
         return _series_op(self, other, 'crosses')
 
     def disjoint(self, other):
-        """Return True for all geometries that are disjoint with *other*, else False"""
+        """Returns a ``Series`` of ``dtype('bool')`` with value ``True`` for
+        each geometry disjoint to `other`.
+
+        An object is said to be disjoint to `other` if its `boundary` and
+        `interior` does not intersect at all with those of the other.
+
+        Parameters
+        ----------
+        other : GeoSeries or geometric object
+            The GeoSeries (elementwise) or geometric object to test if is
+            disjoint.
+        """
         return _series_op(self, other, 'disjoint')
 
     def intersects(self, other):
-        """Return True for all geometries that intersect *other*, else False"""
+        """Returns a ``Series`` of ``dtype('bool')`` with value ``True`` for
+        each geometry that intersects `other`.
+
+        An object is said to intersect `other` if its `boundary` and `interior`
+        intersects in any way with those of the other.
+
+        Parameters
+        ----------
+        other : GeoSeries or geometric object
+            The GeoSeries (elementwise) or geometric object to test if is
+            intersected.
+        """
         return _series_op(self, other, 'intersects')
 
     def overlaps(self, other):
         """Return True for all geometries that overlap *other*, else False"""
         return _series_op(self, other, 'overlaps')
 
     def touches(self, other):
-        """Return True for all geometries that touch *other*, else False"""
+        """Returns a ``Series`` of ``dtype('bool')`` with value ``True`` for
+        each geometry that touches `other`.
+
+        An object is said to touch `other` if it has at least one point in
+        common with `other` and its interior does not intersect with any part
+        of the other.
+
+        Parameters
+        ----------
+        other : GeoSeries or geometric object
+            The GeoSeries (elementwise) or geometric object to test if is
+            touched.
+        """
         return _series_op(self, other, 'touches')
 
     def within(self, other):
-        """Return True for all geometries that are within *other*, else False"""
+        """Returns a ``Series`` of ``dtype('bool')`` with value ``True`` for
+        each geometry that is within `other`.
+
+        An object is said to be within `other` if its `boundary` and `interior`
+        intersects only with the `interior` of the other (not its `boundary` or
+        `exterior`).
+
+        This is the inverse of :meth:`contains`.
```

Comment:
It seems misleading to say that `within` is the inverse of `contains`, without the clarifying comment that follows in the shapely manual that "The expression `a.contains(b) == b.within(a)` always evaluates to True." That is, I think the typical interpretation of inverse would be that `a.contains(b) == ~a.within(b)`, which is not true.

Reply:
Agreed